packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -123,7 +128,8 @@ abstract class CameraPlatform extends PlatformInterface {
   /// The length of the recording can be limited by specifying the [maxVideoDuration].
   /// By default no maximum duration is specified,
   /// meaning the recording will continue until manually stopped.
-  /// The video is returned as a [XFile] after calling [stopVideoRecording].
+  /// With [maxVideoDuration] set the video is returned in a [VideoRecordedEvent]
+  /// in the [onVideoRecordedEvent] stream.
```

Comment:
nit: small update on the description to make it more explanatory.

```suggestion
  /// With [maxVideoDuration] set the video is returned as a [VideoRecordedEvent]
  /// through the [onVideoRecordedEvent] stream when the set duration is reached.
```